internal/controller/ngrok/kubernetesoperator_controller.go

```diff
@@ -51,6 +51,7 @@ import (
 	ngrokv1alpha1 "github.com/ngrok/ngrok-operator/api/ngrok/v1alpha1"
 	"github.com/ngrok/ngrok-operator/internal/controller"
 	"github.com/ngrok/ngrok-operator/internal/ngrokapi"
+	ngrokgo "golang.ngrok.com/ngrok"
```

Comment:
```suggestion
        "github.com/ngrok/ngrok-api-go/v6"
```
The library you are pulling in is for the [ngrok go sdk](https://github.com/ngrok/ngrok-go). We want the [ngrok-api-go](https://github.com/ngrok/ngrok-api-go) lib here for checking API errors.

Reply:
So I asked about this here and I'm confused about which errors we care about. 

https://ngrok.slack.com/archives/C057633V0E7/p1729800605168539?thread_ts=1729800021.427839&cid=C057633V0E7

I think we should chat through this in a huddle so I can grok our ngrok errors :P